tests/ducktape-pr-retry.yml

```diff
@@ -0,0 +1,17 @@
+pass_criteria:
+  zero-drift:
+    mode:
+      dynamic_reliability:
+        compare_with_branch: dev
+        time_window: 2w
+        allowed_drift: 0 # upstream_reliability - current_ci_reliability <= allowed_drift
+        fallback_reliability: 90.0 # when reliability cannot be fetched from branch (e.g. new test)
+retry:
+  default:
+    repeat: 10
+    retry_on:
+      include: # match stacktrace of failure
+        - .*
+      exclude:
```

Comment:
This is probably fine for now but eventually it's going to be a hassle to manage different `include` or `exclude` sections (maybe other sections) for PR and merge when we mostly want them to be the same, across all branches.

So we should think about some mechanism to DRY this, e.g., include a common.yaml while parsing and then yaml anchors+aliases to refer to common sections.

Reply:
Yeah, there is a new bk step defined and will run each time a config is changed (on PRs). For more see this commit https://github.com/redpanda-data/vtools/commit/ae17578e325a36923b8a654deceaf538d535757c

Reply:
\+ it'll parse both the merge and the pr configs

Reply:
added a common file with anchors